### ISSUE TITLE
Buttons need to be labelled consistently

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#112]
URL: https://hcdev.openplus.ca/en and
https://hcdev.openplus.ca/en/search/site

User Agent: Firefox 93.0
Language: Both

Issue Description: The buttons “Find” and “Search” provide the same functionality but are identified differently. They should be identified consistently.  'Search ' is the more universally applied search term.
![image](https://user-images.githubusercontent.com/68117281/137020136-4e00999e-4e08-4e8b-8827-440a8ff9206b.png)

![image](https://user-images.githubusercontent.com/68117281/137020924-7ef0637b-77f0-4044-8277-492ac05c538b.png)

Steps to reproduce: Navigate to both urls and visually scan the buttons.

Expected results: Components that have the same functionality across web application pages, should be identified consistently.

**Comment by ndolter** [comment]
Validated that the button label has now been changed to Search / Recherche.
